Add number separators to query history results

![image](https://github.com/tconbeer/harlequin/assets/705906/5d528c0d-7119-4e84-839d-16a2e5e8ae96)

just a tweak to the f-string here:
https://github.com/tconbeer/harlequin/blob/a78944fafca3c52498b053c473f83f6a1a663d1b/src/harlequin/history.py#L26

Hi @tconbeer , i followed your Walkthrough video, but when i run the  ``` make ``` command i get this error in mypy section

```
=================================================== 148 passed, 3 deselected in 108.40s (0:01:48) ====================================================
mypy
src/harlequin/plugins.py:9:1: error: Cannot find implementation or library stub for module named "importlib_metadata"  [import-not-found]
src/harlequin/plugins.py:9:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
src/harlequin/cli.py:24:1: error: Cannot find implementation or library stub for module named "importlib_metadata"  [import-not-found]
tests/conftest.py:15:1: error: Cannot find implementation or library stub for module named "importlib_metadata"  [import-not-found]
tests/conftest.py:77: error: Unused "type: ignore" comment  [unused-ignore]
tests/conftest.py:84: error: Unused "type: ignore" comment  [unused-ignore]
tests/conftest.py:91: error: Unused "type: ignore" comment  [unused-ignore]
Found 6 errors in 3 files (checked 63 source files)
make: *** [Makefile:6: check] Error 1
```
I tried running : 

```
poetry add importlib_metadata
```
But it is already present in the toml file, can you help me out here 

OS: Ubuntu 22
python : 3.10

--- a/src/harlequin/history.py
+++ b/src/harlequin/history.py
@@ -23,7 +23,7 @@ class QueryExecution:
             result = Text("ERROR", style="bold italic red", justify="right")
         else:
             res = (
-                f"{self.result_row_count:n} "
+                f"{self.result_row_count:,} "
                 f"{'record' if self.result_row_count == 1 else 'records'}"
                 if self.result_row_count
                 else "SUCCESS"

